upcoming_changes/2871.enhancements.rst

```diff
@@ -0,0 +1,4 @@
+* Enhancement of drawing.marker
+  1. Add new marker "arrow" and "ellipse"
+  2. Add support for filled rectangle
+  3. Add "zorder" property for markers
```

Comment:
This should be changed depending on what is decided with my comment about `zorder` above.

Reply:
I removed the 3rd line.